admin-manual/customization/authentication.rst

```diff
@@ -381,6 +381,391 @@ see:
      whose only initial permissions in AtoM will be those of the Authenticated
      :term:`user group`. No password will be stored in AtoM with this method, 
      since the LDAP server handles authentication. This is the default behavior 
-     in AtoM when LDAP is enabled and no user accounts are present. 
+     in AtoM when LDAP is enabled and no user accounts are present.
+
+.. _oidc-enabling:
```

Comment:
Add link to 'oidc-enabling' at top of page (try search for 'ldap-enabling' repo-wide - this may return additional places in the docs we refer to this section from)